requirements.dev.txt

```diff
@@ -8,6 +8,7 @@ types-setuptools==74.0.0.20240831
 types-toml==0.10.8.20240310
 flake8-junit-report==2.1.0
 time-machine==2.15.0
+mypy-baseline==0.7.1
```

Comment:
I think this in the dev deps

Reply:
https://github.com/inmanta/inmanta-dev-dependencies/blob/master/pyproject.toml#L26